src/views/Articles/Posts.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import PostForm from './PostForm';
+import Spinner from '../../components/Articles/Spinner';
+import PostFeed from '../../components/Articles/PostFeed';
+import { getPosts } from '../../redux/actions/postActions';
+
+
+
+class Posts extends Component {
+  componentDidMount() {
+    this.props.getPosts();
+  }
+
+  render() {
+    const { posts, loading } = this.props.post;
+    let postContent;
+
+    if (posts === null || loading) {
+      postContent = <Spinner />;
+    } else {
+      postContent = <PostFeed posts={posts} />;
+    }
+
+    return (
+
+      <div className="col-md-12">
+        <PostForm />
+        {postContent}
+      </div>
```

Comment:
Hello @Amoswachira and @Bakley ,
Please do away with the empty lines ie. from 31 to 37, it makes the code unnecessarily long.

Reply:
@Ogutu-Brian it is still a work in progress the final push will fix all the errors